frontend/components/panels/HighestBidCheckPanel.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title> {{ currentStateAndTitle.title }} </template>
+        <TextBlock v-if="isExplanationsShown">
+            In order to win the auction, participants can bid on the auction. In case someone puts a higher bid on the
+            auction, you receive your MKR back, only losing the transaction fees.
+        </TextBlock>
+
+        <div class="flex justify-between my-3">
+            Current Highest Bid
+            <span v-if="latestBid === null" class="text-gray-400"> There were no bids yet </span>
+            <FormatCurrency v-else :value="latestBid" currency="MKR" />
+        </div>
+
+        <div class="flex justify-end gap-5">
+            <BaseButton
+                class="w-full md:w-80"
+                :type="isUserLatestBidder ? 'secondary' : 'primary'"
+                :disabled="isBidding || isDisabled || !userWalletAddress || isLoading || !!error"
+                :is-loading="isBidding"
+                @click="$emit('restart')"
+            >
+                <span v-if="isBidding"> Bidding... </span>
+                <span v-else-if="auction"
+                    >Bid <FormatCurrency :value="bidAmount" currency="MKR" /> on
+                    <FormatCurrency :value="auction.receiveAmountDAI" currency="DAI"
+                /></span>
+                <span v-else-if="error">Unknown</span>
+                <span v-else>Loading...</span>
```

Comment:
Please use `Unknown` as the "default" case, since otherwise if something is wrong (like when no auction is given) the button will forever be loading, where Unknown is the better case